src/mono/wasi/mono-wasi-driver/driver.c

```diff
@@ -114,15 +114,6 @@ struct WasmSatelliteAssembly_ {
 static WasmSatelliteAssembly *satellite_assemblies;
 static int satellite_assembly_count;
 
-int32_t time(int32_t x) {
-	// In the current prototype, libSystem.Native.a is built using Emscripten, whereas the WASI-enabled runtime is being built
-	// using WASI SDK. Emscripten says that time() returns int32, whereas WASI SDK says it returns int64.
-	// TODO: Build libSystem.Native.a using WASI SDK.
-	// In the meantime, as a workaround we can define an int32-returning implementation for time() here.
-	struct timeval time;
-	return (gettimeofday(&time, NULL) == 0) ? time.tv_sec : 0;
-}
```

Comment:
If this copy of `time` is removed, there's always a build warning like this:

```
wasm-ld: warning: function signature mismatch: time
>>> defined as (i32) -> i32 in ~/dotnet7/packs/Microsoft.NETCore.App.Runtime.Mono.browser-wasm/7.0.0-preview.3.22151.6/runtimes/browser-wasm/native/libSystem.Native.a(pal_random.c.o)
>>> defined as (i32) -> i64 in ~/runtime/artifacts/obj/mono/Wasi.Release/wasi-sdk-12.0/share/wasi-sysroot/lib/wasm32-wasi/libc.a(time.o)
```

Is it possible to leave this strange workaround in place until I sort out building libSystem.Native.a via WASI SDK?

Reply:
Yes, done.